modules/core/src/main/java/org/apache/ignite/spi/tracing/Scope.java

```diff
@@ -63,6 +70,20 @@ public short idx() {
         return idx;
     }
 
+    /**
+     * @return Coordinates without label.
+     */
+    @NotNull public TracingConfigurationCoordinates coordinates() {
```

Comment:
Please don't use `@NotNull` annotation. Our coding guidelines prohibit it https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-@Annotations

Reply:
Fixed